implementation/src/main/java/io/smallrye/health/HealthMessages.java

```diff
@@ -0,0 +1,13 @@
+package io.smallrye.health;
+
+import org.jboss.logging.Messages;
+import org.jboss.logging.annotations.Message;
+import org.jboss.logging.annotations.MessageLogger;
+
+@MessageLogger(projectCode = "SRHCK", length = 5)
+interface HealthMessages {
```

Comment:
Why do you need to have this message in separate interface?

Reply:
I assumed that I have to, according with this [documentation](https://github.com/smallrye/smallrye-parent/wiki/I18N-Messages-and-Logging), but I can be wrong.

Reply:
@ejba thank you for the link, I must have missed that the logging was standardized :)